modAionImpl/src/org/aion/zero/impl/blockchain/AionPendingStateImpl.java

```diff
@@ -441,7 +441,7 @@ public synchronized TxResponse addPendingTransaction(AionTransaction tx) {
     }
 
     public boolean isValid(AionTransaction tx) {
```

Comment:
this method can be a static method?

Reply:
It cannot be changed to static because of the interface for the pending state.

Reply:
okay.